src/components/track/desktop/stats/Stats.tsx

```diff
@@ -47,7 +60,17 @@ const Stats = memo(
       [styles.large]: size === 'large'
     })
 
-    const slice = followeeActions.slice(0, MAX_REPOST_IMAGES)
+    const getFolloweeActionsUsers = useMemo(makeFolloweeActionsUsers, [])
```

Comment:
Why does this have to be a memo? Can you just define `getFolloweeActionUsers` outside the component?

Reply:
I think this is necessary as there will be several `Stats` components grabbing different users, and reselect would cause the selectors to all share the same cache (size of 1) if I didn't make a new one per instance.

> However, when the selector is used in multiple component instances and depends on the component's props, you need to ensure that each component instance gets its own selector instance (see here for a more thorough explanation of why this is necessary):

https://react-redux.js.org/api/hooks#using-memoizing-selectors